qa/acceptance-tests/src/test/java/io/camunda/it/cluster/dynamicnodeid/DynamicNodeIdIT.java

```diff
@@ -64,28 +65,36 @@ public class DynamicNodeIdIT {
           .withoutNodeId()
           .withNodeConfig(
               app ->
-                  app.withAdditionalProperties(
-                      Map.of(
-                          "camunda.data.secondary-storage.type",
-                          "none",
-                          "camunda.cluster.size",
-                          CLUSTER_SIZE,
-                          "camunda.cluster.node-id-provider.type",
-                          "s3",
-                          TASK_ID_PROPERTY,
-                          UUID.randomUUID().toString(),
-                          "camunda.cluster.node-id-provider.s3.bucketName",
-                          BUCKET_NAME,
-                          "camunda.cluster.node-id-provider.s3.leaseDuration",
-                          LEASE_DURATION,
-                          "camunda.cluster.node-id-provider.s3.endpoint",
-                          S3.getEndpoint(),
-                          "camunda.cluster.node-id-provider.s3.region",
-                          S3.getRegion(),
-                          "camunda.cluster.node-id-provider.s3.accessKey",
-                          S3.getAccessKey(),
-                          "camunda.cluster.node-id-provider.s3.secretKey",
-                          S3.getSecretKey())))
+                  app.withProperty(
+                          "camunda.data.secondary-storage.type", SecondaryStorageType.none.name())
+                      .withUnifiedConfig(
+                          cfg -> {
+                            cfg.getData().getSecondaryStorage().setType(SecondaryStorageType.none);
+                            cfg.getCluster().setSize(CLUSTER_SIZE);
+                            cfg.getCluster().getNodeIdProvider().setType(Type.S3);
+                            cfg.getCluster()
+                                .getNodeIdProvider()
```

Comment:
💭 would be nice if we can chain setters on the same class like:
```
s3Config.setLeaseDuration(..).setAccessKey(..)
``` 
If the class does not return `this` when setting them you can change it :+1: